site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -526,6 +526,8 @@
 	
 	private static final String STATE_SITE_ACCESS_INCLUDE = "state_site_access_include";
 
+    private static final String STATE_SITE_ACCESS_PRESENCE_TOOL = "state_site_access_presenceTool";
```

Comment:
I think this file is indented with tabs, rather than spaces. Could you update this please.